apex-app-archetype/src/main/resources/archetype-resources/src/test/resources/log4j.properties

```diff
@@ -1,4 +1,4 @@
-log4j.rootLogger=DEBUG,CONSOLE
+log4j.rootLogger=INFO,CONSOLE
```

Comment:
I would say blacklist what you don't want to see. Your line 21 already changes the com.datatorrent category to info, is that not sufficient? Why change the root logger? User of the archetype should initially see all output of user written code without extra steps.